README.md

```diff
@@ -1,63 +1,77 @@
-# Cumulus
+# Cumulus :cloud:
 
-A set of tools for writing [Polkadot](https://github.com/paritytech/polkadot) parachains that are based on [Substrate](https://github.com/paritytech/substrate).
+A set of tools for writing [Substrate](https://substrate.dev/)-based
+[Polkadot](https://wiki.polkadot.network/en/)
+[parachains](https://wiki.polkadot.network/docs/en/learn-parachains). Refer to the included
+[overview](docs/overview.md) for more details.
 
-It's easy to write blockchains using Substrate, and the overhead of writing parachains' distribution, p2p, database, and synchronization layers is generally high and should be reusable. This project aims to make it easy to write parachains for Polkadot by leveraging the power of Substrate.
+It's easy to write blockchains using Substrate, and the overhead of writing parachains'
+distribution, p2p, database, and synchronization layers should be just as low. This project aims to
+make it easy to write parachains for Polkadot by leveraging the power of Substrate.
 
-Cumulus clouds are shaped sort of like dots and are up in the air, like this project (as it is an initial prototype -- expect a rename when it gets cooler.)
+Cumulus clouds are shaped sort of like dots and are up in the air, like this project (as it is an
+initial prototype -- expect a rename when it gets cooler.)
```

Comment:
I wrote this line in Jan 2019 - I think the name stuck and we can remove this line : ) 

This project doesn't seem "up in the air" anymore

Reply:
Alright I threw a little jazz on it let me know what you think :saxophone: